### ISSUE TITLE
Double clicked of TouchArea in Flickable does not trigger

### ISSUE THREAD

**Issue by FloVanGH** [slint-ui/slint#5586]
If you place a `TouchArea` inside of a `Flickable`  the `double-clicked` callback does not trigger. `clicked` callback works fine.

### Example

```slint
export component Demo {
    Flickable {
        TouchArea {
            width: root.width;
            height: root.height;
            clicked => {
                debug("Clicked");
                rect.background = #000000;
            }

            double-clicked => {
                debug("Double Clicked");
                rect.background = #ffffff;
            }
        }
    }

    rect := Rectangle {
        background: green;
    }
}
```
